src/Ubiquity

```diff
@@ -810,6 +815,17 @@ class Ubiquity {
 				echo ConsoleFormatter::showMessage($msg . "Use <b>Ubiquity serve:stop</b> {$type} to stop it!\n");
 				system('php .ubiquity' . \DS . $serverInfos['start']);
 				break;
+            case 'roadrunner':
+                if (! \file_exists("vendor/lapinskas/roadrunner-ubiquity")) {
+                    system('composer require ' . $serverInfos['require']);
+                }
```

Comment:
To be removed if **checkClass** is filled in

Reply:
Done